chain/client/src/stateless_validation/partial_witness/partial_witness_tracker.rs

```diff
@@ -178,6 +159,7 @@ impl PartialEncodedStateWitnessTracker {
             client_sender,
             epoch_manager,
             parts_cache: LruCache::new(NUM_CHUNKS_IN_WITNESS_TRACKER_CACHE),
```

Comment:
nit. can we rename this to match the other one, eg WITNESS_PARTS_CACHE_SIZE or something?

Reply:
done: 75d8cdf0dc9206e96ddcadb2046d769ba0235451